src/Features/CSharp/Portable/NavigationBar/CSharpNavigationBarItemService.cs

```diff
@@ -2,28 +2,24 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
```

Comment:
git detected this as a move from teh editor layer down to the features layer.  it keeps teh item computation code (though changed to use more modern roslyn patterns), but loses the navigation code.